File: C#/SubmitInstance/ISettings.cs
namespace SubmitInstance
{
	public interface ISettings
	{
		void Load(Stream stream);
		string GetAction { get; }
		string Url { get; }
		string UrlT { get; }
		string NS { get; }
		string SubmitAction { get; }
		string TestSubmitAction { get; } 
	}
}

File: VB/SubmitInstance.Shared/ISettings.vb
Imports System.IO

Public Interface ISettings
	Sub Load(ByVal stream As Stream)
	ReadOnly Property GetAction As String
	ReadOnly Property Url As String
	ReadOnly Property UrlT As String
	ReadOnly Property NS As String
	ReadOnly Property SubmitAction As String
	ReadOnly Property TestSubmitAction As String
End Interface